### DISCUSSION TITLE
v0.13.3

### DISCUSSION THREAD

**Discussion by AsamK** [AsamK/signal-cli#1510] [Releases]
## What's Changed
* Bump h2 from 0.3.24 to 0.3.26 in /client by @dependabot in https://github.com/AsamK/signal-cli/pull/1501
* Document the unit of "start" and "length" for mentions and text styles by @brenns10 in https://github.com/AsamK/signal-cli/pull/1505

## New Contributors
* @brenns10 made their first contribution in https://github.com/AsamK/signal-cli/pull/1505

[CHANGELOG](https://github.com/AsamK/signal-cli/blob/master/CHANGELOG.md#0133---2024-04-19)

**Full Changelog**: https://github.com/AsamK/signal-cli/compare/v0.13.2...v0.13.3

<hr /><em>This discussion was created from the release <a href='https://github.com/AsamK/signal-cli/releases/tag/v0.13.3'>v0.13.3</a>.</em>

**Comment by poeTHEinjector** [comment]
Hello!

Is there a way to configure this CLI with a custom Signal server? I'm looking to connect to my own Signal server instead of the default one.

Thank you!

**Reply by AsamK** [reply]
There's currently no option to configure the Server.
You can however adapt the config in the code and compile signal-cli yourself, see https://github.com/AsamK/signal-cli/issues/722#issuecomment-922338509

**Reply by poeTHEinjector** [reply]
Thank you for the information!
